README.rst

```diff
@@ -541,6 +541,20 @@ Choose from the list of available rules:
   - ``annotations`` (``array``): list of annotations to remove, e.g. ``["author"]``;
     defaults to ``[]``
 
+* **general_phpdoc_tag_rename** [@Symfony]
+
+  Renames PHPDoc tags.
+
+  Configuration options:
+
+  - ``case_sensitive`` (``bool``): whether tags should be replaced only if they have
+    exact same casing; defaults to ``false``
+  - ``fix_annotation`` (``bool``): whether annotation tags should be fixed; defaults
+    to ``true``
+  - ``fix_inline`` (``bool``): whether inline tags should be fixed; defaults to ``true``
+  - ``replacements`` (``array``): a map of tags to replace; defaults to
+    ``['inheritDocs' => 'inheritDoc']``
```

Comment:
`general_*` rule shall have empty default config (meaning they shall not do anything until configured) + must not be part of any ruleset.
`general_*` rules are sth to be configured by end user to match his case, when added to ruleset, they will get overwritten by usage directly added in config file. for that reason, ruleset always use dedicated proxy fixer pointing to configured `general_*` rule instance

Reply:
Default updated.